common/src/main/scala/org/apache/celeborn/common/metrics/source/AbstractSource.scala

```diff
@@ -321,6 +358,22 @@ abstract class AbstractSource(conf: CelebornConf, role: String)
     updateInnerMetrics(sb.toString())
   }
 
+  def recordMeter(nm: NamedMeter): Unit = {
+    val timestamp = System.currentTimeMillis
+    val sb = new StringBuilder
+    val label = nm.labelString
+    sb.append(s"${normalizeKey(nm.name)}Count$label ${nm.meter.getCount} $timestamp\n")
+    sb.append(s"${normalizeKey(nm.name)}MeanRate$label ${nm.meter.getMeanRate} $timestamp\n")
+    sb.append(
+      s"${normalizeKey(nm.name)}OneMinuteRate$label ${nm.meter.getOneMinuteRate} $timestamp\n")
+    sb.append(
+      s"${normalizeKey(nm.name)}FiveMinuteRate$label ${nm.meter.getFiveMinuteRate} $timestamp\n")
+    sb.append(
+      s"${normalizeKey(nm.name)}FifteenMinuteRate$label ${nm.meter.getFifteenMinuteRate} $timestamp\n")
```

Comment:
Refer: https://github.com/apache/spark/blob/667cff76b248c5ea00dd2be2fa9791224053c452/core/src/main/scala/org/apache/spark/metrics/sink/PrometheusServlet.scala#L93-L101

<img width="781" alt="image" src="https://github.com/user-attachments/assets/73e93a1b-31a1-44d7-b6d0-6c65190d3da3">

Reply:
<img width="1188" alt="image" src="https://github.com/user-attachments/assets/8193c12c-5dc9-4783-b64b-6a8449a1bea4">

cc @FMX

Reply:
Finally, we have meters. That's good news.